.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
While you have done a good job setting up the linters :heavy_check_mark: , there are some checks that are not passing. Please run the linters as indicated by this file and make sure there are no linter errors so your project has better readability and looks cleaner :sparkles: 

**Hint:** You can run the linters locally by opening a new terminal in your IDE, copy/pasting the command for each linter (You can find this in this file whenever it says `run:` ) and running it like so:

`$ npx hint .`